games-roguelike/tomenet/files/tomenet-4.7.3-makefile.patch

```diff
@@ -0,0 +1,32 @@
+diff --git a/makefile b/makefile
+index ee56a33..7f3c81f 100644
+--- a/makefile
++++ b/makefile
```

Comment:
can you just check whether the makefile also respects `CPPFLAGS`? This is a user flag that should be respected

Reply:
Done